lib/dockerfile/instruction-parser.ts

```diff
@@ -6,11 +6,12 @@ export {
   getLayersFromPackages,
   getPackagesFromDockerfile,
   instructionDigest,
+  getPackagesFromRunInstructions,
 };
 
 // Naive regex; see tests for cases
 // tslint:disable-next-line:max-line-length
-const installRegex = /\s*(rpm\s+-i|rpm\s+--install|apk\s+((--update|-u|--no-cache)\s+)*add(\s+(--update|-u|--no-cache))*|apt-get\s+((--assume-yes|--yes|-y)\s+)*install(\s+(--assume-yes|--yes|-y))*|apt\s+((--assume-yes|--yes|-y)\s+)*install|yum\s+install|aptitude\s+install)\s+/;
+const installRegex = /(rpm\s+-i|rpm\s+--install|apk\s+((--update|-u|--no-cache)\s+)*add(\s+(--update|-u|--no-cache))*|apt-get\s+((--assume-yes|--yes|-y)\s+)*install(\s+(--assume-yes|--yes|-y))*|apt\s+((--assume-yes|--yes|-y)\s+)*install|yum\s+install|aptitude\s+install)\s+/;
```

Comment:
did you remove the `\s*` on purpose? why?

Reply:
Yep, since there are a lot of tests around this regex, I made an assumption that if it didn't break any existing tests it should be ok.

The reason I removed is that if the leading whitespace isn't remove, this should also support some other cases with instruction from config - for example the `/bin/bash -c` since the `-c` is recognized as a flag. Should work for other flags following that as well. Let me know if that makes sense :)